src/modules/account/submodule/multisig/views/ViewAccountMultisigEditAccount.vue

```diff
@@ -184,35 +185,47 @@ export default {
     const selectedAddresses = ref([]);
     const showAddressError = ref([]);
     const onPartial = ref(false);
-    const isMultisig = ref(true);
+  
     const showCosigners = ref(false);
     const cosignAddress = ref('');
     const selectMainCosign = ref('');
     const selectOtherCosign = ref([]);
 
     // current wallet
     const wallet = walletState.currentLoggedInWallet;
-
     // get account details initialization
-    const acc =  walletState.currentLoggedInWallet.accounts.find(element => element.name ===p.name);
-    const cosigneraddress = cosigner => Address.createFromPublicKey(cosigner, networkState.currentNetworkProfile.network.type).plain().substr(-4)
+    const acc =  computed(()=>{
+      let account
+      if (wallet.accounts.find(element => element.name ===p.name) === undefined && wallet.others.find(element => element.name ===p.name) === undefined){
+        return new WalletAccount("a","b","c","d","e","f")
```

Comment:
what does this code do ? any reason to return it ?

Reply:
when i refresh this page, there are chances that both acc goes undefined, i need this validation to get rid of the error

Reply:
I mean why return a fake WalletAccount, why not just return null ?

Reply:
fixed on my end